src/rbac/application/common/jwt_token_provider_impl.py

```
@@ -45,5 +47,5 @@ def __verify_token(self, token: str) -> Dict[str, Any]:
                 algorithms=[self.algorithm]
             )
             return decoded  # dict 형태로 반환됨
-        except PyJWTError as e:
-            raise ValueError(f"Invalid JWT token: {e}")
+        except PyJWTError:
+            raise ApplicationException(code = CodeEnum.FRS_003, message="Invalid token")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)